src/test/customMatchers.ts

```diff
@@ -1,15 +1,6 @@
 import { ContractTransaction } from "ethers";
 import { getVmError } from "../core/contracts/contract";
 
-declare global {
```

Comment:
Apparently the better way to do this is to add it in a jest.d.ts file and not here.